app/models/explainer.rb

```diff
@@ -71,13 +65,14 @@ def self.update_paragraphs_in_alegre(id, previous_paragraphs_count, timestamp)
       explainer_id: explainer.id
     }
 
+    models_thresholds = Explainer.get_alegre_models_and_thresholds(explainer.team_id).keys
```

Comment:
This could be an instance method, too... and then you woudn't need to pass the `explainer.team_id` argument.

Reply:
No @caiosba as method called from a class method in another place